### ISSUE TITLE
Wrong CustomFieldValue type 

### ISSUE THREAD

**Issue by asafronienkov** [cloudbees-oss/zendesk-java-client#273]
Exception is occur During creating ticket 
org.zendesk.client.v2.ZendeskException: java.lang.IllegalArgumentException: Can not deserialize instance of java.lang.String[] out of VALUE_FALSE token
at [Source: N/A; line: -1, column: -1] (through reference chain: org.zendesk.client.v2.model.Ticket[“custom_fields”]->java.util.ArrayList[2]->org.zendesk.client.v2.model.CustomFieldValue[“value”])

**Comment by cmokbel** [comment]
I Fixed it by adding to Zendesk.java

 mapper.configure(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY, true);
 mapper.configure(SerializationFeature.WRITE_SINGLE_ELEM_ARRAYS_UNWRAPPED, true);


    public static ObjectMapper createMapper() {
        ObjectMapper mapper = new ObjectMapper();
        mapper.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
        mapper.enable(DeserializationFeature.READ_ENUMS_USING_TO_STRING);
        mapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
        mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
        mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
        mapper.setDateFormat(new ISO8601DateFormat());

        **mapper.configure(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY, true);
        mapper.configure(SerializationFeature.WRITE_SINGLE_ELEM_ARRAYS_UNWRAPPED, true);**

        return mapper;
    }

**Comment by aheritier** [comment]
Was fixed by #228 ?
The field is marked with

```
    @JsonFormat(with = {JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY,
            JsonFormat.Feature.WRITE_SINGLE_ELEM_ARRAYS_UNWRAPPED})
```
